tools/test-js-with-node

```diff
@@ -170,6 +170,7 @@ EXEMPT_FILES = make_set(
         "web/src/personal_menu_popover.js",
         "web/src/playground_links_popover.ts",
         "web/src/plotly.js.d.ts",
+        "web/src/pm_conversations.ts",
```

Comment:
You should add some tests verifying the new logic rather than adding this to a coverage exclusion.

Sam with `util.ts`.

A nice little suite of tests really helps with code review for logic like this, because it provides a very clear understanding of which data setup scenarios work.